src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -1111,6 +1113,15 @@ function setTooltipTriggerOnFeedbackPathMenuOptions() {
 $(document).ready(() => {
     prepareInstructorPages();
 
+    $(document).on('click', '#fsDiscardChanges', () => {
+        const okCallback = function () {
```

Comment:
Added arrow function in fe90903. However, the reason I didn't use the arrow function in the first place was that all other `okCallback`s in the same file did not use it. So for consistency, I did not either.